src/App.tsx

```diff
@@ -1,10 +1,73 @@
-/* eslint-disable jsx-a11y/label-has-associated-control */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import { useEffect, useState } from 'react';
+import cn from 'classnames';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { TodoStatusFilter } from './types/TodoStatusFilter';
+import { Notification } from './components/Notifications/Notifications';
+
+const getfilteredTodos = (
+  todos: Todo[],
+  { status }: { status: TodoStatusFilter },
+) => {
+  let filteredTodos = [...todos];
+
+  if (status !== TodoStatusFilter.all) {
+    filteredTodos = filteredTodos.filter(todo => {
+      if (status === TodoStatusFilter.completed) {
```

Comment:
enum values are usually capitalized
```suggestion
      if (status === TodoStatusFilter.Completed) {
```